pages/contact-us.md

```diff
@@ -12,6 +12,25 @@ Below are ways to get in touch with the Apache Accumulo community.
 
 Accumulo uses GitHub issues to track bugs and new features. Visit [How to contribute](/how-to-contribute) for more information.
 
+## Security Issues (CVE)
+
+We strongly encourage reporting potential security issues by privately emailing `private@accumulo.apache.org` or 
+`security@apache.org`
+
+Do not make information about the vulnerability public until it is formally announced by the Accumulo community. 
+That means, for example, that you should not create a public GitHub issue, since those would make the issue public. 
+GitHub pull requests and any messages associated with any commits should not make any reference to the security nature 
+of the commit.
```

Comment:
I added a do not list - I wanted to keep the general process because that mirrors the text from ASF security with Accumulo substituted where appropriate.  I did not want to stray too far from the standard process to show that we are following the ASF policies and eliminate any confusion or conflict.